test/browser/toChildArray.test.js

```diff
@@ -59,7 +59,7 @@ describe('props.children', () => {
 		expect(scratch.innerHTML).to.equal('<div></div>');
 	});
 
-	it('throws an error if given a function child', () => {
+	it.skip('throws an error if given a function child', () => {
```

Comment:
Same story here

Reply:
Does this test cover anything the other one to do with function children doesn't? If not I would suggest to just remove it.